cmd/helm/lint.go

```diff
@@ -60,14 +61,20 @@ func lintCmd(cmd *cobra.Command, args []string) error {
 		return errLintNoChart
 	}
 
-	issues := lint.All(path)
+	linter := lint.All(path)
 
-	if len(issues) == 0 {
+	if len(linter.Messages) == 0 {
 		fmt.Println("Lint OK")
```

Comment:
Even though this code works since `linter.HighestSeverity` is set to 0 by default and now we have hardcoded `ErrorSev`, I'd return here just to make it clear.